module/VuFind/src/VuFind/RateLimiter/RateLimiterManager.php

```diff
@@ -146,6 +149,16 @@ public function check(EventInterface $event): array
             // We have a policy matching the route, so check rate limiter:
             $limiter = ($this->rateLimiterFactoryCallback)($this->config, $policyId, $this->clientIp, $this->userId);
             $limit = $limiter->consume(1);
+            if ($this->config['Policies'][$policyId]['turnstileRateLimiterSettings'] ?? false) {
```

Comment:
Should `$limit->isAccepted()` be checked here before continuing to Turnstile? Or do we always want to consume a Turnstile request too even if the basic rate limiter didn't accept the request?